hera_cal/tests/test_cal_formats.py

```diff
@@ -49,8 +51,8 @@ def test_delayHC(self):
         uv.read_calfits(os.path.join(
             DATA_PATH, 'test_input', 'zen.2457698.40355.xx.HH.uvc.first.calfits'))
         for param in hc:
-            print param
-            print getattr(hc, param).value, getattr(uv, param).value
+           # print param
```

Comment:
Instead of commenting these out, we should just get rid of them.

Reply:
ya, that was a holdout from debugging...